repose-aggregator/components/filters/keystone-v2-filter/src/main/resources/META-INF/schema/config/keystone-v2-common.xsd

```diff
@@ -100,7 +103,9 @@
                     </xs:documentation>
                 </xs:annotation>
                 <xs:complexType>
+                    <!-- TODO: Does the authorization filter need a default-tenant-quality and roles-tenant-quality? -->
                     <xs:attribute name="default-tenant-quality" type="QualityType" default="0.9" use="optional"/>
+                    <!-- TODO: Should this be renamed to matching-tenant-quality? Not backwards compatible. -->
```

Comment:
This is probably just something we want to do eventually. I'm not sure we want to do anything about this right now, though.

Reply:
default tenant quality and matching tenant quality are two different concerns

Reply:
Oh, the TODO is actually above the line for the `uri-tenant-quality` which is now being used not just for the URI tenant but for the matching tenant. This line was referring to the `uri-tenant-quality` attribute, not the `default-tenant-quality` attribute. GitHub just chooses to give you the commented on line and the line above it, but not the line below...